Assets/js/custom-objects.js

```diff
@@ -46,6 +46,7 @@ CustomObjects = {
             'autocomplete': valueField.attr('autocomplete'),
             'value': valueField.attr('value')
         };
+        let fieldType = selectedField.attr('data-field-type');
```

Comment:
```suggestion
        const fieldType = selectedField.attr('data-field-type');
```
Let's use `const` for variables that are never written to.